ui/src/app/pages/login/account-creation/account-creation-institution.tsx

```diff
@@ -0,0 +1,330 @@
+              </i>
+            </label>
+            <div style={{...styles.text, fontSize: 14}}>
+              Your institution will be notified that you have registered using your institutional credentials.
+            </div>
+            <Dropdown
+                data-test-id='institution-dropdown'
+                style={{width: '50%', minWidth: '600px'}}
+                options={institutions.map(inst => ({'value': inst.shortName, 'label': inst.displayName}))}
+                value={institutionShortName}
+                onChange={(e) => {
+                  this.updateAffiliationValue('institutionShortName', e.value);
+                  // Clear out any existing values for role when the institution changes.
+                  this.updateAffiliationValue('institutionalRoleEnum', undefined);
+                  this.updateAffiliationValue('institutionalRoleOtherText', undefined);
+                }}/>
+            {this.state.dataLoadError &&
+            <ErrorDiv data-test-id='data-load-error'>
+              An error occurred loading the institution list. Please try again or contact
+              <a href='mailto:support@researchallofus.org'>support@researchallofus.org</a>.
+            </ErrorDiv>
+            }
+            <div style={{marginTop: '.5rem'}}>
+              <a href={'https://www.researchallofus.org/apply/'} target='_blank' style={{color: colors.accent}}>
+              Don't see your institution listed?
+              </a>
+            </div>
+            <TextInputWithLabel containerStyle={{marginTop: '1rem', width: null}}
+                                value={contactEmail}
+                                inputId='contact-email'
+                                inputName='contactEmail'
+                                labelContent={<div>
+                                  <label style={{...styles.text, fontWeight: 600}}>
+                                    Your institutional email address
+                                  </label>
+                                  <div style={{...styles.text, fontSize: 14}}>
+                                    This will be the primary email contact for your new account.
+                                  </div>
+                                </div>}
+                                invalid={this.state.emailFailedValidation}
+                                onBlur={() => this.validateContactEmailInline()}
+                                onChange={email => this.updateContactEmail(email)}/>
+            {this.state.emailFailedValidation &&
+              <ErrorDiv data-test-id='invalid-email-error'>
+                Error: email address is invalid
+              </ErrorDiv>
+            }
+            <div style={{marginTop: '.5rem'}}>
+              <label style={{...styles.text, fontWeight: 600, marginTop: '1rem'}}>
+                Which of the following best describes your role?
+                <i style={{...styles.publiclyDisplayedText, marginLeft: '0.2rem'}}>
```

Comment:
Would it be possible to fold the `marginLeft: '0.2rem'` into a style object? This is the second place it is used.

Reply:
I'll punt on this one – the margins in this form are pretty haphazard, I wouldn't really know how to name such a style object.

Reply:
Sounds reasonable. TOL, since we have this a couple places in the account creation area, we could make a sub component for this, but I am okay punting on that.